src/components/hosts/HostRowDetail.tsx

```diff
@@ -165,12 +165,7 @@ export const HostDetail: React.FC<HostDetailProps> = ({
   }
   bmcAddress = bmcAddress || DASH;
 
-  let virtualText = DASH;
-  if (inventory.systemVendor?.virtual) {
-    virtualText = 'Virtual machine';
-  } else if (inventory.systemVendor?.virtual !== undefined) {
-    virtualText = 'Bare metal';
-  }
+  const virtualText = getHardwareTypeText(inventory);
```

Comment:
Nit: `virtualText` is a bit confusing

```suggestion
  const hardwareType = getHardwareTypeText(inventory);
```